SmartPlaces.Facilities/lib/IngestionManager.Mapped/src/MappedGraphIngestionProcessor.cs

```diff
@@ -99,7 +99,13 @@ private async Task UpdateOutputSiteAsync(JsonElement siteElement, CancellationTo
 
             await GetPlacesAsync(twins, relationships, siteElement, null, null);
 
-            var siteDtId = siteElement.GetProperty("id").ToString();
+            if (!siteElement.TryGetProperty("id", out var idProp))
+            {
+                Logger.LogInformation("No SiteId found...");
+                return;
```

Comment:
With this short circuit maybe the method should return a Task<bool> for isSuccess dimension because on line79 the metrics would indicate that it was added.